### ISSUE TITLE
Namespace support and SOCKET_*

### ISSUE THREAD

**Issue by xeor** [nathantsoi/vue-native-websocket#60]
When I want to use namespaces, how can I handle the SOCKET_* mutations?
I got namespaced messages to work, but I am getting messages like `[vuex] unknown mutation type: SOCKET_ONOPEN`.. Where do I store the `SOCKET_*` mutations?

**Comment by NikitaKA** [comment]
I've faced with the same problem. It seems that websocket protocol doesn't support data send on connection (https://stackoverflow.com/questions/12192321/is-it-possible-to-send-a-data-when-a-websocket-connection-is-opened).

**Comment by UglyHobbitFeet** [comment]
Yeah, I had to do a similar hack. I took the passToStoreHandler method shown in the readme, and changed let target = eventName.toUpperCase() into let target = 'myStorePath/' + eventName.toUpperCase(). Far from ideal, but it works for my particular scenario.
